### ISSUE TITLE
.recordings not found

### ISSUE THREAD

**Issue by pepa65** [gphalkes/tilde#22]
At the current state of all the involved repos, when following the build recipe in the readme, the build ends with:
```
==== Executing make -C src for tilde ====
make: Entering directory '/home/pp/git/gphalkes/tilde/src'
[CXX] fileautocompleter.cc
[CXX] filebuffer.cc
[CXX] fileeditwindow.cc
[CXX] fileline.cc
[CXX] filestate.cc
[CXX] filewrapper.cc
[CXX] main.cc
[CXX] openfiles.cc
[CXX] option.cc
[CXX] util.cc
[CXX] dialogs/attributesdialog.cc
[CXX] dialogs/encodingdialog.cc
[CXX] dialogs/highlightdialog.cc
[CXX] dialogs/openrecentdialog.cc
[CXX] dialogs/selectbufferdialog.cc
[CXX] dialogs/optionsdialog.cc
[LD] .objects/edit
find: ‘.recordings/’: No such file or directory
find: ‘.recordings/’: No such file or directory
make: Leaving directory '/home/pp/git/gphalkes/tilde/src'
```
This might be harmless, because `tilde/src/.objects/edit` still seems to work correctly. Maybe the `find` in the Makefile should be made conditional upon the presence of `.recordings`.

**Comment by gphalkes** [comment]
Fixed by commit 8dc84f832f7b91e22dc40e21bd58dff4e32db500

**Comment by pepa65** [comment]
Fix confirmed, thanks!
